### ISSUE TITLE
The header's design is… aggressive

### ISSUE THREAD

**Issue by haydn** [sketchbook-js/website#3]
I think we'll want to tone it down a bit.

**Comment by haydn** [comment]
@Mxchaeltrxn I had a bit of a play around. How does this feel to you?

![website-git-darkmode-sketchbook-js vercel app_](https://user-images.githubusercontent.com/27211/124340990-a0be4e00-dbfc-11eb-83ba-1c1579917756.png)

It's not super fancy, but it's functional.

**Comment by Mxchaeltrxn** [comment]
@haydn It feels a bit plain but it definitely works!

Though maybe I've been looking at the old logo for too long so I'm used to the splash of colour 😅

**Comment by haydn** [comment]
Maybe this instead?

![image](https://user-images.githubusercontent.com/27211/124372714-bcdeef80-dccf-11eb-9a1a-9afc63b3bc6e.png)

The other thing I was considering is something along these lines:

![image](https://user-images.githubusercontent.com/27211/124373004-2e1fa200-dcd2-11eb-9442-bf6b5adcd57f.png)

IMO, the name and the logo feel a bit disjointed in that one.

**Comment by Mxchaeltrxn** [comment]
@haydn I agree with you when you say in the third one, the logo and name are a bit disjointed. 

Of the three, I like the first one you showed me the most.

Question:
If we don't have pink in the background, does that mean the favicon has to change too? If it does, doesn't it become really hard to recognise the logo? (I'm just imagining it based on the current favicon)

![image](https://user-images.githubusercontent.com/34886045/124374278-477a1b80-dcdd-11eb-8e2e-6e2e279f4a66.png)
